public/style/main.css

```diff
@@ -168,8 +326,11 @@ label {
 
 
 .isochroneSlider {
-  max-height : 33px;
-  overflow : hidden;
+  height : 30px;
+  width: 300px;
```

Comment:
dont set the width of something if it's contrary to the width of it's parent - this creates overflow issues which you then went ahead and explicitly set to auto which means the browser will not know what to do